### ISSUE TITLE
Doesn't work with shmck:angular2@2.0.6

### ISSUE THREAD

**Issue by 30suns** [ShMcK/ng2-socially-tutorial#1]
If you use the latest version of shmck:angular2@2.0.6 this tutorial will fail at step 4. The errors are hard for me to trace because I'm completely new to meteor, angular and nodejs. I diffed this repo against my local one and the only difference was this repo uses shmck:angular2@2.0.4. Once I made that change, to my local version it worked. Hope that's helpful!

``` javascript
ReferenceError: require is not defined
var angular2_1 = require('angular2/angular2');
```

Here are some things that show up in firebug:

``` javascript
EXCEPTION: Error during instantiation of PartiesForm!.
ORIGINAL EXCEPTION: TypeError: Rx.Subject is not a constructor
EXCEPTION: TypeError: componentRef is undefined
```

**Comment by ShMcK** [comment]
Thanks, I'll look into what the breaking change might be and update the tutorial.

It's likely due to a change between alpha 35 & alpha 36 versions.

**Comment by ShMcK** [comment]
You're right. Step 4 breaks when FORM_DIRECTIVES are introduced. I'm still not sure what has changed, but I'll keep looking.

**Comment by ShMcK** [comment]
[An issue](https://github.com/angular/angular/issues/3987) is posted in the Angular repo showing the same error starting from alpha-36.

**Comment by 30suns** [comment]
Does this resolve with an updated version of System.js as mentioned in the linked issue reports?

**Comment by nazarhussain** [comment]
Had any one find the solution for this issue?

**Comment by ShMcK** [comment]
Not sure. The issue occurs at step 4.5, specifically when `FORM_DIRECTIVES` are imported. There are other working examples using the same Angular2 version and similar code.

`shmck:angular2` will be changed over to `urigo:angular2-meteor` in the near future, which should resolve these issues in the tutorial.

**Comment by nazarhussain** [comment]
@ShMcK Thanks for the response. How long we should wait to use `urigo:angular2-meteor`?

**Comment by ShMcK** [comment]
It's nearly ready. You can try it now by cloning the repo into your packages directory.
